chalice/deploy/planner.py

```diff
@@ -589,51 +596,21 @@ def _plan_cloudwatchevent(self, resource):
             # You need to remove targets (which have IDs)
             # before you can delete a rule.
             models.RecordResourceValue(
-                resource_type='cloudwatch_event',
+                resource_type=resource.resource_type,
```

Comment:
So I think it makes sense to keep this hardcoded as `cloudwatch_event`. Specifically in the sweeper, we look for `cloudwatch_event` as the resource type when we try to delete the rule. I think right now (I need to double check), the sweeper will not delete the rule resource. Otherwise, we need to go back to the sweeper and add these resources as strings to look for when deleting the rules.

Reply:
good catch, reverted back to cloudwatch_event static string.